### ISSUE TITLE
notation of 'coordinate_1_based'

### ISSUE THREAD

**Issue by vjcitn** [LiNk-NY/AnnotationHubShiny#5]
value 0 is presumably denoting FALSE?

<img width="1331" alt="ahsx" src="https://user-images.githubusercontent.com/4624066/132990587-35a71593-4212-4064-98e2-5eb5b79d3aa2.png">

**Comment by LiNk-NY** [comment]
Yes, it looks like this field is documented in the vignette as logical but somewhere along the line it changes type to integer...
https://bioconductor.org/packages/release/bioc/vignettes/AnnotationHub/inst/doc/CreateAHubPackage.html
```r
library(AnnotationHub)
ah <- AnnotationHub()
table(ah$coordinate_1_based)
# 
#     0     1 
# 25849 33986 
class(ah$coordinate_1_based)
# [1] "integer"
```
